src/styles/colors.css

```diff
@@ -35,12 +35,13 @@
   --palette-warning-dark: #ffd15a;
   --palette-warning-base: #ffdf8d;
   --palette-warning-light: #ffedc0;
+
   /* Warning interactions */
   --palette-warning-hover: #ffd874;
 
   /* Neutral */
   --palette-neutral-xx-dark: #282624;
-  --palette-neutral-x-dark: #333333;
+  --palette-neutral-x-dark: #333;
```

Comment:
I'm not quite sure why we needed to do this but it works so ok ;)

Reply:
It is part of the standard Stylelint rules from the "popular" ruleset. I figured we might as well keep the  set of rules until we feel really strongly against the rules.